modules/optimization/test/tests/executioners/constrained/inequality/main_auto_adjoint.i

```diff
@@ -0,0 +1,77 @@
+# This tests that a linear and constant function can be scaled in
+# two separate functionNeumannBCs both applied to the same sideset using
+# two parsed functions.  The scale of the linear and constant functions
+# are being parameterized.
+
+[Optimization]
+[]
+
+[OptimizationReporter]
+  type = OptimizationReporter
+  parameter_names = 'left'
+  num_values = '2'
+  initial_condition = '10 10'
+  lower_bounds = '0'
+  upper_bounds = '1000'
+
+  inequality_names = 'ineq'
+
+  measurement_file = 'measurementData.csv'
```

Comment:
I would put the measurements into the file.  I also would include all four measurements, I think a single measurement would be underconstrained.  
```suggestion
  measurement_points = '0.2 0.2 0
                        0.8 0.6 0
                        0.2 1.4 0
                        0.8 1.8 0'
  measurement_values = '207 204 185 125'
```
You'll need to update these with synthetic measurements for your problem.